css/css-text/text-transform/reference/text-transform-full-size-kana-001-ref.html

```diff
@@ -0,0 +1,72 @@
+<body>
+<p class="instructions">Test passes if both characters in each pair match. If you are missing a font glyph for a character, ignore that pair, but report which characters were ignored.</p>
+<div class="test">
+  <span title="U+3041">&#x3042; &#x3042;</span>
+  <span title="U+3043">&#x3044; &#x3044;</span>
+  <span title="U+3045">&#x3046; &#x3046;</span>
+  <span title="U+3047">&#x3048; &#x3048;</span>
+  <span title="U+3049">&#x304A; &#x304A;</span>
+  <span title="U+3095">&#x304B; &#x304B;</span>
+  <span title="U+3096">&#x3051; &#x3051;</span>
+  <span title="U+3063">&#x3064; &#x3064;</span>
+  <span title="U+3083">&#x3084; &#x3084;</span>
+  <span title="U+3085">&#x3086; &#x3086;</span>
+  <span title="U+3087">&#x3088; &#x3088;</span>
+  <span title="U+308E">&#x308F; &#x308F;</span>
+  <span title="U+30A1">&#x30A2; &#x30A2;</span>
+  <span title="U+30A3">&#x30A4; &#x30A4;</span>
+  <span title="U+30A5">&#x30A6; &#x30A6;</span>
+  <span title="U+30A7">&#x30A8; &#x30A8;</span>
+  <span title="U+30A9">&#x30AA; &#x30AA;</span>
+  <span title="U+30F5">&#x30AB; &#x30AB;</span>
+  <span title="U+31F0">&#x30AF; &#x30AF;</span>
+  <span title="U+30F6">&#x30B1; &#x30B1;</span>
+  <span title="U+31F1">&#x30B7; &#x30B7;</span>
+  <span title="U+31F2">&#x30B9; &#x30B9;</span>
+  <span title="U+30C3">&#x30C4; &#x30C4;</span>
+  <span title="U+31F3">&#x30C8; &#x30C8;</span>
+  <span title="U+31F4">&#x30CC; &#x30CC;</span>
+  <span title="U+31F5">&#x30CF; &#x30CF;</span>
+  <span title="U+31F6">&#x30D2; &#x30D2;</span>
+  <span title="U+31F7">&#x30D5; &#x30D5;</span>
+  <span title="U+31F8">&#x30D8; &#x30D8;</span>
+  <span title="U+31F9">&#x30DB; &#x30DB;</span>
+  <span title="U+31FA">&#x30E0; &#x30E0;</span>
+  <span title="U+30E3">&#x30E4; &#x30E4;</span>
+  <span title="U+30E5">&#x30E6; &#x30E6;</span>
+  <span title="U+30E7">&#x30E8; &#x30E8;</span>
+  <span title="U+31FB">&#x30E9; &#x30E9;</span>
+  <span title="U+31FC">&#x30EA; &#x30EA;</span>
+  <span title="U+31FD">&#x30EB; &#x30EB;</span>
+  <span title="U+31FE">&#x30EC; &#x30EC;</span>
+  <span title="U+31FF">&#x30ED; &#x30ED;</span>
+  <span title="U+30EE">&#x30EF; &#x30EF;</span>
+  <span title="U+FF67">&#xFF71; &#xFF71;</span>
+  <span title="U+FF68">&#xFF72; &#xFF72;</span>
+  <span title="U+FF69">&#xFF73; &#xFF73;</span>
+  <span title="U+FF6A">&#xFF74; &#xFF74;</span>
+  <span title="U+FF6B">&#xFF75; &#xFF75;</span>
+  <span title="U+FF6F">&#xFF82; &#xFF82;</span>
+  <span title="U+FF6C">&#xFF94; &#xFF94;</span>
+  <span title="U+FF6D">&#xFF95; &#xFF95;</span>
+  <span title="U+FF6E">&#xFF96; &#xFF96;</span>
+</div>
+<!--Notes:
+Tip: To identify the characters where differences occur, in order to report problem characters, either mouse over to reveal a tooltip, or copy and paste the sequence into a tool such as <a href='http://r12a.github.io/uniview/' target='_blank'>UniView</a> or the <a href='http://r12a.github.io/apps/conversion/' target='_blank'>Unicode Conversion Tool</a>.
```

Comment:
This tip is not correct. copy&paste should copy the original untransformed character, so Uniview won't help. (same comment in other tests).

Spec reference for that claim:
https://drafts.csswg.org/css-text-3/#text-transform-property
> This property transforms text for styling purposes. It has no effect on the underlying content, and must not affect the content of a plain text copy & paste operation.